manage.py

```diff
@@ -26,6 +28,11 @@ def init_db(archive=None):
     More information about populating a PostgreSQL database efficently can be
     found at http://www.postgresql.org/docs/current/static/populate.html.
     """
+    print('Creating user and a database...')
+    exit_code = subprocess.call('sudo -u postgres psql < ' + os.path.join('admin', 'sql', 'create_db.sql'), shell=True)
```

Comment:
It will not create a database. I can add an argument that will allow to skip database creation if that's something that will be helpful.